src/logic/PlayerController.cpp

```diff
@@ -835,8 +835,7 @@ void PlayerController::onUpdateByInput(float deltaTime)
             }
             else
             {
-                // This happens when NPC enters area that should not be accessible
```

Comment:
For this particular instance, if could very well be that there is simply no type assigned in the level files, which is something we can't do anything about. We should check the modkit, however, so we can confirm that.

Logging something every frame is a performance problem on windows builds, since the windows console is much slower than any linux terminal.